libcontainer/intelrdt/intelrdt.go

```diff
@@ -169,10 +171,17 @@ type IntelRdtManager struct {
 	Config *configs.Config
 	Id     string
 	Path   string
+	Type   string
```

Comment:
How to pass parameter "IntelRdtManager.Type" to a container which is being created or loaded?
Do we need to configure "IntelRdtManager.Type" in IntelRdtFs(), LinuxFactory.Create(), LinuxFactory.Load() and loadFactory()?